core/frontend/package.json

```diff
@@ -118,6 +119,15 @@
             "./src/tile/internal.ts"
           ]
         }
+      ],
+      "@itwin/public-extension-exports": [
+        "error",
+        {
+          "releaseTags": [
+            "public",
+            "preview"
+          ]
+        }
```

Comment:
Do you want the output file to be false here the same way it is in `core/common`?